### PR TITLE
Update docs home page

### PR DESCRIPTION
**Pull request by legalosLOTR**

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -26,25 +26,19 @@ IAM, Kubernetes, Elastic Load Balancing etc -- that's what Opta does!
...
-Opta is designed for folks who are not Infrastructure or Devops experts but still want to build amazing,
-scalable, secure Infra in the cloud. Majority of Opta's users are early stage startups who use it for their
-dev/staging/production environments.
+We are confident it can drastically reduce the cloud complexity and devops headaches of most fast moving 
+organizations. It is already being used by companies - big and small :)
 
-If you'd like to try it out or have any questions - feel free to join our [Slack](https://slack.opta.dev/)!
+To read more about the vision behind Opta, check out this [blog post](https://blog.runx.dev/infrastructure-as-code-for-everyone-7dad6b813cbc).
 
-## What you get with Opta
```
Comment (juandiegopalomino) [inline content/en/_index.md:37]: you sure you wanna torch this section?

Comment (legalosLOTR) [reply content/en/_index.md:37]: Yeah - this information is already covered in the README.

**Review by juandiegopalomino — APPROVED** [review]

File: content/en/_index.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ menu:
...
 ## What is Opta?
 
-Opta is a new kind of Infrastructure-As-Code framework where you work with high-level constructs
-instead of getting lost in low level cloud configuration. Imagine just being able to say that you want
-an autoscaling docker container that talks to a RDS database - instead of figuring out the details of VPC,
-IAM, Kubernetes, Elastic Load Balancing etc -- that's what Opta does!
+Opta is an Infrastructure-As-Code framework where you work with high-level constructs instead of getting lost
+in low level cloud configuration. Opta gives you a vast library of modules that you can connect together to
+build your ideal Infrastructure stack. Best of all, Opta uses Terraform under the hood - so you're never locked in.
+You can always write custom Terraform or even take the Opta generated Terraform and go your own way!
```
Comment (RemyDeWolf) [inline content/en/_index.md:17]: There is also the idea that a user might generate Terraform with opta and repeat this process, may be with some automation. Go your own ways, makes it sound that opta is no longer needed.
May something like:
Use opta to generate your Terraform stack to provision your infrastructure yourself.

File: content/en/_index.md [hunk 3]
```diff
@@ -26,25 +26,19 @@ IAM, Kubernetes, Elastic Load Balancing etc -- that's what Opta does!
   </iframe>
 </p>
 
-## Who is it for?
+## Why use Opta
+Infrastructure as code (IaC) has rapidly become the standard for provisioning and managing Infrastructure and 
+for the right reasons! But the leading IaC tools are still complicated to use and require deep Cloud/Infrastructure
+expertise. Opta was conceptualized to help address this complexity. Opta is a simpler IaC framework with best practices
+built-in. It enables users to set up automated, scalable and secure infrastructure without being a cloud expert or 
+spending days figuring out cloud minutiae!
```
Comment (RemyDeWolf) [inline content/en/_index.md:34]: After reading this paragraph I understand your idea of having some graphics to convey our message. Something to consider

File: content/en/_index.md [hunk 4]
```diff
@@ -26,25 +26,19 @@ IAM, Kubernetes, Elastic Load Balancing etc -- that's what Opta does!
...
 
-Opta is designed for folks who are not Infrastructure or Devops experts but still want to build amazing,
-scalable, secure Infra in the cloud. Majority of Opta's users are early stage startups who use it for their
-dev/staging/production environments.
+We are confident it can drastically reduce the cloud complexity and devops headaches of most fast moving 
+organizations. It is already being used by companies - big and small :)
```
Comment (RemyDeWolf) [inline content/en/_index.md:37]: We should start gathering testimony and join articles in one place to illustrate this section

File: content/en/_index.md (hunk 3 above)
Comment (legalosLOTR) [reply content/en/_index.md:34]: yeah that's in my list of todos!

File: content/en/_index.md (hunk 2 above)
Comment (legalosLOTR) [reply content/en/_index.md:17]: Hmm... I think we need to expand the custom-terraform module docs (https://docs.opta.dev/reference/aws/environment_modules/custom-terraform/) into a whole "Opta + Terraform" guide. And that's where we could include this info. I'll make a note of this.

**Pull request merged**